.eslintrc.js

```diff
@@ -1 +1,5 @@
 module.exports = require('eslint-config-wolox-node');
+
+module.exports.rules.indent = ['off']
```

Comment:
why this?

Reply:
There is a conflict between this rule and prettify